rs/nervous_system/common/build_metadata/src/lib.rs

```diff
@@ -27,13 +27,13 @@ const BUILD_INFO_FORMAT: &str = r#"
 /// There are some unusual requirements. Basically, calling crates must be ready
 /// to use the build-info crate. More specifically, calling crates must do the
 /// following:
-///   1. List the following in the [dependencies] section of their Cargo.toml:
-///     a. build-info
-///     b. candid
-///     c. dfn_candid
-///     d. dfn_core
-///   2. List build-info-build in the [build-dependencies] section of their Cargo.toml.
-///   3. Have a ./build.rs where main calls build_info::build_script() (either directly or indirectly).
+/// 1. List the following in the [dependencies] section of their Cargo.toml:
+///     * build-info
+///     * candid
+///     * dfn_candid
+///     * dfn_core
+/// 2. List build-info-build in the [build-dependencies] section of their Cargo.toml.
+/// 3. Have a ./build.rs where main calls build_info::build_script() (either directly or indirectly).
```

Comment:
This is tangential.

Also, I do not see how this is an improvement. Seems like moving deck chairs on the Titanic.

Also, do you really want to show up in git blame?

At first, I was going to say "Do as you see fit", but as I look at more of this PR, I see that there is a fair amount of comment reformatting. Because the amount of such changes is "substantial", I'm thinking that we should rethink these changes. Maybe I'm missing some context for why you are making these changes in this PR?

Reply:
The changes are needed as the new clippy version detects that the generated MD can be broken,  [doc_lazy_continuation](https://rust-lang.github.io/rust-clippy/master/index.html#/doc_lazy_continuation). 

No, I don't mind that the changes show up in the git blame history, as it was part of what's needed to upgrade the compiler version.

Reply:
I did not realize these comment changes satisfy clippy.